### PR TITLE
feat: update import service

### PR DESCRIPTION
**Pull request by hetao92**
mod: remove importer and update docs

### CODE REVIEW

File: app/assets/modules/Import/Tasks/Import.tsx [hunk 1]
```diff
@@ -118,10 +120,32 @@ class Import extends React.Component<IProps, IState> {
...
+      // statusStoped、statusProcessing、statusNotExisted、statusAborted, statusFinished
+      if (result && result.taskStatus !== 'statusProcessing') {
+        service.finishImport({ taskId });
+        clearTimeout(this.checkTimer);
+      } else {
+        this.checkTimer = setTimeout(this.checkIfFinished, 2000);
+      }
+    } else {
+      message.warning(errMsg);
+    }
```
Comment (huaxiabuluo) [inline app/assets/modules/Import/Tasks/Import.tsx]: exception logic codes is quite fewer

```ts
if (code !== 0) {
  message.warning(errMsg);
  return;
}

const result = data.results?.[0];

if (result?.taskStatus !== 'statusProcessing') {
  service.finishImport({ taskId });
  clearTimeout(this.checkTimer);
} else {
  this.checkTimer = setTimeout(this.checkIfFinished, 2000);
}
```

File: app/assets/modules/Import/Tasks/Import.tsx [hunk 2]
```diff
@@ -118,10 +120,32 @@ class Import extends React.Component<IProps, IState> {
     });
     if (errCode === 0) {
       this.logTimer = setTimeout(this.readlog, 2000);
+      this.checkTimer = setTimeout(this.checkIfFinished, 2000);
     }
     trackEvent('import', 'import_data', 'start');
   };
 
+  checkIfFinished = async () => {
```
Comment (jerryliang64) [inline app/assets/modules/Import/Tasks/Import.tsx]: checkIfFinished => checkIsFinished is more accurate

File: app/assets/modules/Import/Tasks/Import.tsx [hunk 3]
```diff
@@ -118,10 +120,32 @@ class Import extends React.Component<IProps, IState> {
...
+    }
+
+    const result = data.results?.[0];
+
+    if (result?.taskStatus !== 'statusProcessing') {
+      service.finishImport({ taskId });
+      clearTimeout(this.checkTimer);
+    } else {
+      this.checkTimer = setTimeout(this.checkIfFinished, 2000);
+    }
```
Comment (jerryliang64) [inline app/assets/modules/Import/Tasks/Import.tsx]: you can also avoid the `else` nested via returing early when task is finished

File: app/assets/modules/Import/Tasks/Import.tsx [hunk 4]
```diff
@@ -194,7 +218,12 @@ class Import extends React.Component<IProps, IState> {
           </Button>
           <Button
             className="import-again"
-            onClick={() => this.props.stopImport({ taskId })}
+            onClick={() =>
+              this.props.stopImport({
+                taskID: taskId,
```
Comment (jerryliang64) [inline app/assets/modules/Import/Tasks/Import.tsx:223]: can keep the same style? taskID => taskId?

File: app/assets/store/models/import.ts [hunk 5]
```diff
@@ -285,25 +285,31 @@ export const importData = createModel({
...
-          taskId,
+          taskId: data[0],
           isImporting: true,
         });
       } else {
-        message.error(errMsg);
+        message.error(message);
       }
-      return errCode;
+      return code;
```
Comment (jerryliang64) [inline app/assets/store/models/import.ts:300]: use return to avoid the `else` nested

File: app/assets/modules/Import/Tasks/Import.tsx (hunk 4 above)
Comment (hetao92) [reply app/assets/modules/Import/Tasks/Import.tsx:223]: > can keep the same style? taskID => taskId?
its defined in gateway

**Review by jerryliang64 — APPROVED** [review]
LGTM

**Pull request merged**